build.sbt

```diff
@@ -476,13 +470,21 @@ lazy val scaldingHRaven = module("hraven").settings(
       exclude("com.twitter.common", "application-module-log")
       exclude("com.twitter.common", "application-module-stats")
       exclude("com.twitter.common", "args")
-      exclude("com.twitter.common", "application"),
+      exclude("com.twitter.common", "application")
+      // Adding this dependencies explicitly because otherwise they are broken
+      exclude("com.twitter", "util-registry_2.10")
+      exclude("com.twitter", "util-core_2.10")
+      exclude("com.twitter", "util-jvm_2.10"),
     "org.apache.hbase" % "hbase" % hbaseVersion,
     "org.apache.hbase" % "hbase-client" % hbaseVersion % "provided",
     "org.apache.hbase" % "hbase-common" % hbaseVersion % "provided",
     "org.apache.hbase" % "hbase-server" % hbaseVersion % "provided",
     "org.slf4j" % "slf4j-api" % slf4jVersion,
-    "org.apache.hadoop" % "hadoop-client" % hadoopVersion % "provided"
+    "org.apache.hadoop" % "hadoop-client" % hadoopVersion % "provided",
+
+    "com.twitter" % "util-registry_2.10" % "6.34.0",
+    "com.twitter" % "util-core_2.10" % "6.34.0",
+    "com.twitter" % "util-jvm_2.10" % "6.34.0", 
```

Comment:
can we make a val holding this version constant for this?

Also, fixing to a particular scala version here is really scary. That sounds like a great way for runtime errors if someone tries `2.11` for this artifact.

If it only runs for 2.10, we should have the build filter this out when scalaVersion is not 2.10.

Reply:
Good question. I believe currently hraven is compiled only against scala 2.10, hence this. By making this deps explicit I made behaviour to be the same as before (resolve changed and it started to fail with resolving to non existent version).

I can see 2 alternatives:
- remove module completely
- don't add any explicit dependencies here, or make them provided

Reply:
Decided to remove this deps